README.md

```diff
@@ -5,6 +5,8 @@
 
 Drone plugin for deploying to Cloud Foundry
 
+See [DOCS.md](DOCS.md) for `.drone.yml` usage
```

Comment:
The main entry point for the plugins is the plugin index, on the index you always see DOCS.md for every plugin and no README.md